docker-compose.yml

```diff
@@ -25,3 +31,15 @@ services:
     tty: true
     volumes:
       - ./.env:/home/node/app/.env
+
+  redis:
+    image: redis:latest
+    container_name: redis_cache
+    ports:
+      - '6379:6379'
+    volumes:
+      - redis-data:/data
+    restart: always
+
+volumes:
+  redis-data:
```

Comment:
Is always a good practice to leave a blank line at the end of the file, so is cleaner to append new content at the end.